src/aws-resource-mappings.ts

```diff
@@ -95,350 +83,6 @@ export function mapToAwsResource(
-                },
-                options,
-            );
-        }
-        case 'AWS::EC2::Route':
-            return new aws.ec2.Route(
-                logicalId,
-                {
-                    routeTableId: props.routeTableId,
-                    carrierGatewayId: props.carrierGatewayId,
-                    destinationCidrBlock: props.destinationCidrBlock,
-                    destinationIpv6CidrBlock: props.destinationIpv6CidrBlock,
-                    egressOnlyGatewayId: props.egressOnlyInternetGatewayId,
-                    gatewayId: props.gatewayId,
-                    localGatewayId: props.localGatewayId,
-                    natGatewayId: props.natGatewayId,
-                    networkInterfaceId: props.networkInterfaceId,
-                    transitGatewayId: props.transitGatewayId,
-                    vpcEndpointId: props.vpcEndpointId,
-                    vpcPeeringConnectionId: props.vpcPeeringConnectionId,
-                },
-                options,
-            );
-        case 'AWS::EC2::NatGateway':
-            return new aws.ec2.NatGateway(
-                logicalId,
-                {
-                    subnetId: props.subnetId,
-                    allocationId: props.allocationId,
-                    connectivityType: props.connectivityType,
-                    tags: tags(props.tags),
-                },
-                options,
-            );
-        case 'AWS::ApplicationAutoScaling::ScalableTarget': {
-            const target = new aws.appautoscaling.Target(
-                logicalId,
-                {
-                    maxCapacity: props.maxCapacity,
-                    minCapacity: props.minCapacity,
-                    resourceId: props.resourceId,
-                    roleArn: props.roleArn,
-                    scalableDimension: props.scalableDimension,
-                    serviceNamespace: props.serviceNamespace,
-                },
-                options,
-            );
-            props.ScheduledActions?.map(
-                (action: any) =>
-                    new aws.appautoscaling.ScheduledAction(
-                        logicalId + action.scheduledActionName,
-                        {
-                            resourceId: target.resourceId,
-                            scalableDimension: target.scalableDimension,
-                            scalableTargetAction: action.scalableTargetAction,
-                            schedule: action.schedule,
-                            serviceNamespace: target.serviceNamespace,
-                            startTime: action.startTime,
-                            endTime: action.endTime,
-                            timezone: action.timezone,
-                            name: action.scheduledActionName,
-                        },
-                        options,
-                    ),
-            );
-            return target;
-        }
         // IAM
         case 'AWS::IAM::Policy': {
```

Comment:
Curious this is really not supported by Cloud Control API or is there another reason this maps to AWS classic?

Reply:
It's not fully supported in CCAPI (missing READ/LIST) so it is not supported by aws-native.